docs/source/reference/numpysupported.rst

```diff
@@ -330,11 +330,10 @@ The following top-level functions are supported:
 * :func:`numpy.concatenate`
 * :func:`numpy.convolve` (only the 2 first arguments)
 * :func:`numpy.copy` (only the first argument)
-* :func:`numpy.corrcoef` (only the 3 first arguments, requires NumPy >= 1.10 and
-  SciPy >= 0.16; extreme value handling per NumPy 1.11+)
+* :func:`numpy.corrcoef` (only the 3 first arguments)
```

Comment:
does this still require SciPy?

Reply:
good catch. deleted too much

Reply:
I think it will need SciPy as `corrcoef` calls `cov` calls `dot` which needs BLAS which requires SciPy cython pointers. There's probably a load of unlisted dependency chains like this.